securedrop-admin

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env python2
+"""
+SecureDrop Admin Toolkit.
+
+For use by administrators to install, maintain, and manage their SD
+instances.
+"""
+
+
+import argparse
+import subprocess
+import os
+import sys
+import getpass
+
+SD_DIR = os.path.dirname(os.path.realpath(__file__))
+ANSIBLE_PATH = os.path.join(SD_DIR, "./install_files/ansible-base")
+SITE_CONFIG = os.path.join(ANSIBLE_PATH, "group_vars/all/site-specific")
+VENV_DIR = os.path.join(SD_DIR, ".venv")
+VENV_ACTIVATION = os.path.join(VENV_DIR, 'bin/activate_this.py')
+
+def error(msg, level='warning', exit=True):
+    """ Stupid wrapper function to normalize warnings """
+    print("{} - {}".format(level.upper(), msg))
+    if exit:
+        sys.exit(1)
+
+def activate_venv():
+    """Use to activate the local virtualenv"""
+    try:
+        execfile(VENV_ACTIVATION, dict(__file__=VENV_ACTIVATION))
+    except FileNotFoundError:
+        error("Pre-requisites not in place, re-run command with 'setup'",
+                level='error')
+
+def sdconfig(args):
+    """Configure SD site settings"""
+    activate_venv()
+    subprocess.call(['./configure'], cwd=ANSIBLE_PATH, env=os.environ.copy())
+
+def envsetup(args):
+    """Install SD pre-reqs"""
+    # virtualenv doesnt exist? Install dependencies and create
+    if not os.path.exists(VENV_ACTIVATION):
+        subprocess.call(['sudo', 'su', '-c', 
+                        "apt-get update && \
+                        apt-get install -y \
+                        python-virtualenv \
+                        python-pip \
+                        ccontrol \
+                        virtualenv \
```

Comment:
`virtualenv` is not available for Trusty: https://packages.ubuntu.com/search?suite=trusty&searchon=names&keywords=virtualenv . I think `python-virtualenv`, included above, contains everything we need and this line should be removed.